libethereum/ChainParams.cpp

```diff
@@ -115,7 +115,7 @@ ChainParams ChainParams::loadConfig(
     setOptionalU256Parameter(cp.durationLimit, c_durationLimit);
 
     if (params.count(c_chainID))
-        cp.chainID = int(fromBigEndian<u256>(fromHex(params.at(c_chainID).get_str())));
+        cp.chainID = uint64_t(fromBigEndian<u256>(fromHex(params.at(c_chainID).get_str())));
```

Comment:
Should we maybe do a static_cast instead (since static_casts are both easier to visually identify in code and to search for)?

Reply:
Yes, I just did the same as surrounding code, but I'll change it.